packages/blocks/src/components/drag-handle.ts

```diff
@@ -128,6 +128,9 @@ export class DragHandle extends LitElement {
   protected firstUpdated() {
     this.setAttribute('draggable', 'true');
     this.style.display = 'none';
+    this._indicator = <DragIndicator>(
+      document.querySelector('affine-drag-indicator')
+    );
```

Comment:
Use the query API in lit.

```ts
@query('affine-drag-indicator')
_indicator!: DragIndicator
```

Reply:
I guess `@query` only works in the same component?

Reply:
I can't get `affine-drag-indicator` this way as it's outside of DragHandle now (globally)